libraries/AP_HAL_Linux/RCOutput_Bebop.cpp

```diff
@@ -283,6 +283,28 @@ void RCOutput_Bebop::_play_sound(uint8_t sound)
     _dev->get_semaphore()->give();
 }
 
+void RCOutput_Bebop::play_note(uint8_t pwm, uint16_t period, uint16_t duration)
```

Comment:
> Maybe RCOutput_Bebop is a bad name for that

RCOutput_BLDC I think would be a better name?



> About i2c message :
> 
> if pwm is < 3, then pwm refers to a predefined song stored in BLDC.
> if pwm is > 3, then pwm is really a pwm and the following arguments (period and duration are read)

@mothacehe 
Could you add a comment there with this information?